### PR TITLE
Write down your SRP page fixes

### LINKED ISSUE #13457: Write down your SRP page
- [ ]  The tip "Write down and store in multiple secret places" tip is there twice. Remove one instance.

- [ ] "Store in a bank vault" is redundant since we also say "Store in a safe-deposit box." remove it.*

- [ ] Should we keep "Save in a password manager" or have we decided that's not actually a safe tip?*

- [ ] There is no way to hide the SRP again after revealing, we should provide this option.*

*requires design input (cc @rachelcope)

### PR DESCRIPTION
**Pull request by filipsekulic**
Fixes: https://github.com/MetaMask/metamask-extension/issues/13457

Screenshot: 

Screenshot 2022-02-23 at 09 45 48

Screencast:

https://user-images.githubusercontent.com/92310504/156144002-029e4aee-5329-4da0-8bf2-f3c410f1395e.mov

### CODE REVIEW

**Comment by mirjanaKukic** [comment]
Verified by QA

**Review by darkwing — DISMISSED** [review]

File: ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js [hunk 1]
```diff
@@ -80,21 +77,39 @@ export default function RecoveryPhrase({ secretRecoveryPhrase }) {
...
-              onClick={() => {
-                handleCopy(secretRecoveryPhrase);
-              }}
-              icon={copied ? null : <Copy size={20} color="#3098DC" />}
-              className="recovery-phrase__footer--copy--button"
-              type="link"
-            >
-              {copied ? t('copiedExclamation') : t('copyToClipboard')}
-            </Button>
+          <div className="recovery-phrase__footer--copy-and-hide">
```
[the remark this answers is not in the document]
Comment (filipsekulic) [reply ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js]: Fixed in: b090fe4585b5248eeb1ef6233dd7701ad667ef69

**Review by jpuri — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -1381,6 +1381,9 @@
   "hide": {
     "message": "Hide"
   },
+  "hideSeedPhrase": {
+    "message": "Hide the Seed phrase"
```
[the remark this answers is not in the document]
Comment (AndreySttar) [reply app/_locales/en/messages.json:1385]: S

File: ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AndreySttar) [reply ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js]: Y

**Comment by AndreySttar** [comment]
b090fe4585b5248eeb1ef6233dd7701ad667ef69

File: app/_locales/en/messages.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (filipsekulic) [reply app/_locales/en/messages.json]: Fixed in: 89e82d9b54e553974dc264304c834aad8eb91f76

**Review by adonesky1 — APPROVED** [review]
LGTM

**Review by jpuri — APPROVED** [review]

**Review by adonesky1 — CHANGES_REQUESTED** [review]
Sorry just looped back and noticed possible redundancy?

File: ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js [hunk 3]
```diff
@@ -25,6 +25,8 @@ export default function RecoveryPhrase({ secretRecoveryPhrase }) {
   const t = useI18nContext();
   const [copied, handleCopy] = useCopyToClipboard();
   const [phraseRevealed, setPhraseRevealed] = useState(false);
+  const [hiddenPhrase, setHiddenPhrase] = useState(false);
```
[the remark this answers is not in the document]
Comment (adonesky1) [reply ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js:28]: Ok makes sense.

**Review by adonesky1 — APPROVED** [review]

**Pull request merged**